layers/ntp.go

```diff
@@ -1,6 +1,6 @@
 //******************************************************************************
```

Comment:
Remove these first two lines as well, here and in the other file.

IANAL, so I just do what worked in the past, and that did :)